frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -201,10 +203,23 @@ function ReaderFooter:init()
         padding = 0,
         padding_bottom = self.bottom_padding,
     }
-    self.footer_container = BottomContainer:new{
-        dimen = Geom:new{ w = 0, h = self.height*2 },
-        self.footer_content,
-    }
```

Comment:
I never understood the why of these double BottomContainer, and why one had self.height*2  :)
It was introduced in 6938b543, I hope it does not matter with the touch zone now after many other changes since 2014 :) and that DTAP_ZONE_MINIBAR works just as expected.
(BottomContainer does centering by default, so it acted as the centerer I think - might as well use a CenterContainer as you did for clarity.)

Reply:
I also don't understand why double BottomContainer wa used :)
Touch zones should work because we have separate method:

https://github.com/koreader/koreader/blob/570180196b70b7fc1be502790652400fce2f8927/frontend/apps/reader/modules/readerfooter.lua#L244-L272